extension-base/ctpAzureWrapper.js

```diff
@@ -0,0 +1,77 @@
+// Azure
+
+module.exports = {
+  ctpAzureWrapper: function(fn) {
+    return function(context, req) {
+      fn(req.body, ctpAzureResponse(context), context.log);
+    }
+  }
+}
+
+function ctpAzureResponse(context) {
+  return {
+    pass: passAzure(context),
```

Comment:
I would call this `create*` it's a pattern we took on to indicate that something is closed over in this instance the `context`. It "shows" a bit better that the `pass` still needs to be invoked.

Reply:
yeah, I thought of mentioning that, but it would have been too nitpicky #neuroticswede